test/Concurrency/actor_isolation.swift

```diff
@@ -1467,3 +1467,22 @@ class SGA_MADirect: MADirect {
   // directly-nonisolated vs overridden-MainActor = nonisolated
   nonisolated override func method2() { beets_ma() } // expected-error {{call to main actor-isolated global function 'beets_ma()' in a synchronous nonisolated context}}
 }
+
+// Actor isolation allows capture
+extension MyActor {
+  func testNonSendableCaptures(sc: SomeClass) {
+    Task {
+      _ = self
+      _ = sc
+
+      Task { [sc,self] in
+        _ = self
+        _ = sc
+
+        Task {
+          _ = sc // expected-warning{{capture of 'sc' with non-sendable type 'SomeClass' in a `@Sendable` closure}}
+        }
+      }
```

Comment:
This is weird actually... are you saying this 3rd Task does NOT execute on the actor because does not capture self? That's insanely brittle isn't it? Isn't the model we explain that if "syntactically created inside an actor" then runs on actor...? 😕 We're not doing a good job explaining the actual semantics of the implicit executor inheritance tbh :~

Reply:
If you don't capture `self`, we shouldn't capture it on your behalf. At most, this could be a warning.